config/prow_common.mk

```diff
@@ -56,42 +58,17 @@ get-cluster-credentials:
 unset-cluster-credentials:
 	$(UNSET_CONTEXT)
 
-.PHONY: update-prow-config update-prow-job-config update-prow-plugins update-all-boskos-deployments update-boskos-resource update-almost-all-cluster-deployments update-single-cluster-deployment update-prow test update-testgrid-config confirm-master
+.PHONY: update-prow-config update-all-boskos-deployments update-boskos-resource update-almost-all-cluster-deployments update-single-cluster-deployment test update-testgrid-config confirm-master
 
 # Update prow config
 update-prow-config: confirm-master
 	$(SET_CONTEXT)
-	kubectl create configmap config --from-file=config.yaml=$(PROW_CONFIG) --dry-run --save-config -o yaml | kubectl apply -f -
-	$(UNSET_CONTEXT)
-
-# Update all prow job configs
-update-prow-job-config: confirm-master
-	$(SET_CONTEXT)
-ifndef SKIP_CONFIG_BACKUP
-	$(eval OLD_YAML_CONFIG := $(shell mktemp))
-	$(eval NEW_YAML_CONFIG := $(shell mktemp))
-	$(eval GCS_DEST := $(PROW_CONFIG_GCS)/config-$(shell date '+%Y_%m_%d_%H:%M:%S').yaml)
-	@kubectl get configmap job-config -o jsonpath="{.data['config\.yaml']}" 2>/dev/null > "${OLD_YAML_CONFIG}"
-	@gsutil cp "${OLD_YAML_CONFIG}" "${GCS_DEST}" > /dev/null
-	@cp "$(PROW_JOB_CONFIG)" "${NEW_YAML_CONFIG}"
-	@echo "# FROM COMMIT: $(shell git rev-parse HEAD)" >> "${NEW_YAML_CONFIG}"
-else
-	$(eval NEW_YAML_CONFIG := $(PROW_JOB_CONFIG))
-endif
-# We'll have to use `kubectl replace` here because `kubectl apply` will add the whole original configmap to the
-# `last-applied-configuration` annotation, but k8s has a maxmium 256kb limit for it, see https://github.com/kubernetes/kubectl/issues/712
-	kubectl create configmap job-config --from-file=config.yaml=$(NEW_YAML_CONFIG) --dry-run -o yaml | kubectl replace configmap job-config -f -
-	$(UNSET_CONTEXT)
-ifndef SKIP_CONFIG_BACKUP
-	diff "${OLD_YAML_CONFIG}" "${NEW_YAML_CONFIG}" --color=auto || true
-	@echo "Inspect uploaded config file at: ${NEW_YAML_CONFIG}"
-	@echo "Old config file saved at: ${GCS_DEST}"
-endif
-
-# Update prow plugins
-update-prow-plugins: confirm-master
-	$(SET_CONTEXT)
-	kubectl create configmap plugins --from-file=plugins.yaml=$(PROW_PLUGINS) --dry-run --save-config -o yaml | kubectl apply -f -
+	python3 <(curl -sSfL https://raw.githubusercontent.com/istio/test-infra/master/prow/recreate_prow_configmaps.py) \
```

Comment:
This is from istio/test-infra master branch, Is this stable enough?

Reply:
It should be, but I can change it to use the specific commit like `https://github.com/istio/test-infra/blob/48d88083edbccd5e7ca037dd9b9f2b6d19f5359a/prow/recreate_prow_configmaps.py` if you think it's more preferable, but I guess we will need to change it again soon as this script can be moved to `oss-test-infra` to be shared.

Reply:
I guess it's ok to stick with master branch, depends on how active it's been updated